Packs/CheckpointFirewall/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CheckPoint Firewall v2
+- Moved non-`Client` methods into the client, for better proxy support. 
+- Better error handling
```

Comment:
@dorschw - These notes are more dev-oriented. The notes should be more aligned with what the customer is experiencing. Try to think how they might experience the change made - you can look at some best practices here: https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices

Reply:
Also stay aligned with the formats in the docs please

Reply:
done, can you take another look please?